npm/private/npm_import.bzl

```diff
@@ -457,7 +465,7 @@ def _download_and_extract_archive(rctx):
     # so we use tar here which takes a --strip-components N argument instead of rctx.download_and_extract
     untar_args = ["tar", "-xf", _TARBALL_FILENAME, "--strip-components", str(1), "-C", _EXTRACT_TO_DIRNAME, "--no-same-owner", "--no-same-permissions"]
 
-    if repo_utils.is_linux(rctx):
+    if _is_gnu_tar(rctx):
         # Some packages have directory permissions missing the executable bit, which prevents GNU tar from
         # extracting files into the directory. Delay permission restoration for directories until all files
         # have been extracted. We assume that any linux platform is using GNU tar and has this flag available.
```

Comment:
nit: this last sentence doesn't belong here anymore, since we sometimes follow this conditional branch on MacOS